doc/dev_guide/c_style_guide_new.rst

```diff
@@ -0,0 +1,1046 @@
+-------------------------------------------------------------------------------
+                                C Style Guide
+-------------------------------------------------------------------------------
+
+We use Git for revision control. The latest development is happening in the
+default branch (currently ``master``). Our git repository is hosted on GitHub,
+and can be checked out with ``git clone git://github.com/tarantool/tarantool.git``
+(anonymous read-only access).
+
+If you have any questions about Tarantool internals, please post them on the
+`developer discussion list <https://groups.google.com/forum/#!forum/tarantool>`_
+or on `StackOverflow <https://stackoverflow.com/questions/tagged/tarantool>`_.
+Additionally, some engineers are always present on #tarantool channel on
+irc.freenode.net.
```

Comment:
If we don't care about atomicity of the changes, I would delete the google group and irc.freenode.net mentioning. The latter does not even work.